plugins/connectrpc/go/v1.11.0/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/connectrpc/go
```

Comment:
Chose to call the plugin `connectrpc/go` for less stuttering and symmetry with `protocolbuffers/go` and `grpc/go`.

Reply:
Also parity with https://buf.build/grpc/go, so 👍